### ISSUE TITLE
[HOLD - N7 Account Settings] Workspace settings - The input field covers with save button

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#11187]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to URL https://staging.new.expensify.com/
and login with any account
2. Go to Settings->Workspace->General settings
3. Focus the name input field

## Expected Result:
The input fields and the save button must not overlap

## Actual Result:
The Save button can overlap with the default currency input field or the name input field

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Mobile Web (Android)
- Web (desktop)
- Desktop
- iOS app

**Version Number:** v1.2.4-1
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/93399543/192121248-26c15593-4dee-4404-8037-09ab24ac2e4e.mp4

**Expensify/Expensify Issue URL:** 
**Issue reported by:** Applause internal team
**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by sketchydroide** [comment]
I could not replicate on iOS but the GH only mentions Android, that would be a bit weird, as I'm useing chrome on iOS an they should be identical.
Still different platforms might be different.
Going to ask in slack to see if anyone can replicate it.

**Comment by sketchydroide** [comment]
I would say this could be an external though, if we are able to replicate it.

**Comment by sketchydroide** [comment]
in all cases my sugestion it for the save button to never overlap, and if needed. make it part of the scrollable area.

**Comment by sketchydroide** [comment]
@kavimuru can you please remove the Offline/Onlline part of the issue, this is trully just one issue, the offline or online parts just makes it more or less obvious.

I think it should be:

## Action Performed:
1. Go to URL https://staging.new.expensify.com/
and login with any account
2. Go to Settings->Workspace->General settings
3. Put on the name input field

## Expected Result:
The input fields and the save button must not overlap

## Actual Result:
The Save button can overlap with the default currency input field or the name input field

**Comment by sketchydroide** [comment]
Setting this as an external.

I think the excpectations are that the save button goes up, but never goes over the input fields.
If there is no enough space then the entire form can scroll.

**Comment by trjExpensify** [comment]
@iwiznia @shawnborton @JmillsExpensify - this problem becomes obsolete when we update the `General settings` page to be push-to-page. I'm not sure it's worth fixing this in light of that?

Also, @sketchydroide, the behaviour is as intended on the iOS app it seems: 

![IMG_8531](https://user-images.githubusercontent.com/16232057/192774405-29457cfa-1c71-4db4-8044-d0a0dcfc9e39.PNG)

On iOS mWeb (safari) it is not: 

![IMG_8533](https://user-images.githubusercontent.com/16232057/192775070-64a290b0-6dec-4903-bd82-1e63c559ed1c.PNG)

**Comment by sketchydroide** [comment]
> I agree. We're going to fix this real soon for Account Settings re-write. I say we leave it until then.

Ok setting it on Hold and Weekly, removing external and all the other things

**Comment by sketchydroide** [comment]
Yeah that would work @shawnborton.
Do we want to fix it now, or wait for the refactor?
I don't think the priority on getting this fixed is that high...

**Comment by shawnborton** [comment]
Yeah, I think it's okay if we wait.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (Second week)

**Comment by sketchydroide** [comment]
I think it's still reproducible. But I think we are still waiting, I think we can move this to Monthly while we wait.

**Comment by trjExpensify** [comment]
Ohh. @shawnborton with the brand updates for WAQ, this will get taken care of right?

**Comment by shawnborton** [comment]
I'm not sure how this one will change with the branding updates.

But separately, I just tested this on my iPhone, and it works perfectly. Maybe we can close this one out? 

![IMG_4123](https://user-images.githubusercontent.com/2319350/199077973-6042ac1d-1a92-4fd8-88bc-95e239d5207b.PNG)

**Comment by trjExpensify** [comment]
Was that iOS mWeb? https://github.com/Expensify/App/issues/11187#issuecomment-1282610043

**Comment by shawnborton** [comment]
Ah good point - I just tried on mWeb and it's still broken.

**Comment by sketchydroide** [comment]
We still need to decide if we fix now or wait for the branding update.
This should be easy enough for an external to do it. But not worth it if the page is getting worked in a short time as we might a well do it then.
This said, I don't think the priority on this is that high, sure it's annoying, but no like it makes the app unusable also It's only on tiny web screens.

**Comment by trjExpensify** [comment]
Yeah, I think we all agree this waits for push to page. I just thought that the "held" reason was probably now with releasing the branding updates to the workspace editor, whereby this would be reworked as push to page.

**Comment by JmillsExpensify** [comment]
Gotcha, I think the latest is that the branding updates are not touching push-to-page. Or rather, we are not implementing push-to-page as a requirement for completing the branding updates, so I do kind of think that this is something that depends on N7. Maybe we add it to that project and close this?

**Comment by trjExpensify** [comment]
Can we get a vid, @mvtglobally to make sure the steps are correct? I can reliably reproduce it still on mWeb safari (iPhone 13 Pro).

**Comment by sketchydroide** [comment]
@mvtglobally any updates on @trjExpensify request?

**Comment by sketchydroide** [comment]
bump @mvtglobally 🙏🏼

**Comment by sketchydroide** [comment]
I can't replicate this myself anymore, and the page looks very different as well.
Closing this.
